src/app/core/guards/view/view-exist.guard.ts

```diff
@@ -49,7 +49,7 @@ export class ViewExistGuard implements CanActivate {
     }
 
     return this.store.select(selectViewsLoaded).pipe(
-      skipWhile((loaded) => !loaded),
+      filter((loaded) => !loaded),
```

Comment:
The value needs to be negated in order to keep similar behaviour to `skipWhile`.

I have removed this guard in my other PR. Was `skipWhile` operator causing some bug so I should change it in a new guard like you did?

Reply:
Maybe in this situation using `skipWhile` and `filter` is the same. But in `effects` using `skipWhile` over `filter` is not a good idea...